{
  try {
    this.pool.returnObject(this.object);
  }
 catch (  Exception exc) {
    throw new IOException(exc);
  }
}
